db/pushdb.go

```diff
@@ -145,6 +146,37 @@ func (f *pushDatabaseOpts) AddPushServiceProviderToService(service string,
 	}
 	f.dblock.Lock()
 	defer f.dblock.Unlock()
+
+	/*
+	 * #198:
+	 * 2017 Victor Lang
```

Comment:
nit: Authorship of individual lines of code changes over time, and can be found via `git log` and `git blame`, and authorship of specific sections isn't tracked elsewhere in the code.